packages/backend/src/api/controllers/status/discovery/props/dashboardContracts/getDescription.ts

```diff
@@ -0,0 +1,11 @@
+import { EthereumAddress } from '@l2beat/shared'
+
+import { DiscoveryConfig } from '../../../../../../core/discovery/DiscoveryConfig'
+
+export function getDescription(
+  field: string,
+  address: EthereumAddress,
+  config: DiscoveryConfig,
+): string | undefined {
```

Comment:
```suggestion
export function getDescription(
  config: DiscoveryConfig,
  address: EthereumAddress,
  field: string,
): string | undefined {
```
It is a nit pick, but I think we should have a consistent approach to arguments order

Reply:
I agree